nflex_connector_utils/service_offering.py

```diff
@@ -0,0 +1,30 @@
+from . import Resource
+
+
+class ServiceOffering(Resource):
+    """
+        A representation of a server
```

Comment:
Also, can you update docs/api.rst to include the new file? Otherwise API docs won't render. Try make doc & open doc/_build/html/index.html